assets/base/i18n/en.json

```diff
@@ -807,12 +818,19 @@
   "post_info.guest": "Guest",
   "post_info.system": "System",
   "post_message_view.edited": "(edited)",
+  "post_priority.button.acknowledge": "Acknowledge",
   "post_priority.label.important": "IMPORTANT",
   "post_priority.label.urgent": "URGENT",
+  "post_priority.picker.apply": "Apply",
   "post_priority.picker.beta": "BETA",
-  "post_priority.picker.label.important": "Important",
-  "post_priority.picker.label.standard": "Standard",
-  "post_priority.picker.label.urgent": "Urgent",
+  "post_priority.picker.cancel": "Cancel",
+  "post_priority.picker.label.important": "",
+  "post_priority.picker.label.persistent_notifications": "",
+  "post_priority.picker.label.persistent_notifications.description": "",
+  "post_priority.picker.label.request_ack": "",
+  "post_priority.picker.label.request_ack.description": "",
+  "post_priority.picker.label.standard": "",
+  "post_priority.picker.label.urgent": "",
```

Comment:
Empty

Reply:
wat?! how?! thanks @enahum

Reply:
I believe you used the t function, that pick up the if but not the text, so you need to edit the file manually to include the text

Reply:
@enahum yep they got deleted here, https://github.com/mattermost/mattermost-mobile/pull/7142/commits/0d12f72d69ed5854a59037112d26e6b272ccc89a#diff-f5a1cd7d23d8039cabddd344e8251de5e48703cbe05d619fd24d0add2e76faec

which doesn't make 100% sense to me, since I had the text in, it shouldn't be deleted, maybe the order change deleted the text?

Reply:
I'll use `defineMessage`

Reply:
can't use `defineMessages`, it won't be picked by `i18n-extract`.